go/vt/wrangler/traffic_switcher.go

```diff
@@ -156,107 +127,27 @@ func (ts *trafficSwitcher) ForAllTargets(f func(source *workflow.MigrationTarget
 func (wr *Wrangler) getCellsWithShardReadsSwitched(ctx context.Context, targetKeyspace string, si *topo.ShardInfo, tabletType string) (
 	cellsSwitched, cellsNotSwitched []string, err error) {
 
-	cells, err := wr.ts.GetCellInfoNames(ctx)
+	ttype, err := topoproto.ParseTabletType(tabletType)
```

Comment:
If you follow my suggestion from above, here we can:
* Rename to tabletType to be tabletTypeStr
* Just this variable stays `ttype`.